.github/workflows/end2end-test-playwright.yml

```diff
@@ -54,7 +54,7 @@ jobs:
               if: always()
               with:
                   name: failures-artifacts
-                  path: artifacts
```

Comment:
Why does this currently not work? This should support directories by default.

https://github.com/actions/upload-artifact

First location is set here: https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/playwright.config.ts#L11
Second location is set here: https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/playwright.config.ts#L24

Still the same artifacts directory so may be better to figure out why success is not being uploaded.

Reply:
We don't want the directory to be uploaded when the tests are successful. The problem is that Playwright will generate the `storage-state` directory inside `artifacts` too, no matter if the tests have flaky results or not. But that's not useful for debugging when there are no flaky tests. So the solution is to only upload the `test-results` directory, which will only exist if there are failed/flaky results.

Reply:
Makes sense now. 👍